internal/services/healthcare/healthcare_dicom_resource_test.go

```diff
@@ -119,78 +105,109 @@ func (r HealthCareDicomResource) basic(data acceptance.TestData) string {
 %s
 
 resource "azurerm_healthcare_dicom_service" "test" {
-  name         = "dicom%d"
+  name         = "dicom%[2]s"
   workspace_id = azurerm_healthcare_workspace.test.id
-  location     = "%s"
+  location     = azurerm_resource_group.test.location
   depends_on   = [azurerm_healthcare_workspace.test]
+
+  lifecycle {
+    ignore_changes = [
+      "cors",
+      "storage"
+    ]
+  }
 }
-`, r.template(data), data.RandomIntOfLength(10), data.Locations.Primary)
+`, r.template(data), data.RandomString)
 }
 
 func (r HealthCareDicomResource) complete(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 %s
 
+resource "azurerm_storage_account" "test" {
+  name                     = "mystorageaccount%[2]s"
```

Comment:
Minor, but we follow the naming convention `acctest{resource-abbreviation}{randomstring-or-integer}` for the resources we provision in the acceptance tests.
```suggestion
  name                     = "acctestsa%[2]s"
```

I've called out a few below but please make sure to update all resource names in the test configs to follow the naming pattern

Reply:
Got it, updated the names in the test